src/VisualStudio/Core/Def/HACK_ThemeColorFixer.cs

```diff
@@ -110,6 +135,38 @@ private void DeprioritizeStringClassification(IClassificationFormatMap formatMap
             }
         }
 
+        private void DeprioritizeCommentClassification(
+            IClassificationFormatMap formatMap,
+            string typeName,
+            ref int commentIndex)
+        {
+            var classificationType = _classificationTypeRegistryService.GetClassificationType(typeName);
+            if (classificationType is null)
+                return;
+
+            // Don't need to do anything if the classification type is already ordered before comments.
+            var index = formatMap.CurrentPriorityOrder.IndexOf(classificationType);
+            if (index < commentIndex)
+                return;
+
+            formatMap.BeginBatchUpdate();
```

Comment:
>formatMap.BeginBatchUpdate(); [](http://example.com/codeflow?start=12&length=29)

Isn't this going to do many BatchUpdates since this is called in a loop?

Reply:
yes.  let me lift that out for sure!